ocs_ci/deployment/deployment.py

```diff
@@ -803,10 +803,12 @@ def deploy_ocs_via_operator(self, image=None):
                 interfaces.add(config.ENV_DATA["multus_cluster_net_interface"])
             worker_nodes = get_worker_nodes()
             node_obj = ocp.OCP(kind="node")
-            for node in worker_nodes:
-                for interface in interfaces:
-                    ip_link_cmd = f"ip link set promisc on {interface}"
-                    node_obj.exec_oc_debug_cmd(node=node, cmd_list=[ip_link_cmd])
+            platform = config.ENV_DATA.get("platform").lower()
+            if platform == "baremetal":
```

Comment:
We want to skip if baremetal right?

```suggestion
            if platform != "baremetal":
```